client/indexer-db/src/models/paymentstream.rs

```diff
@@ -0,0 +1,31 @@
+use bigdecimal::num_bigint::BigUint;
+use diesel::prelude::*;
+
+use crate::{schema::paymentstream, DbConnection};
+
+#[derive(Debug, Queryable, Insertable, Selectable)]
+#[diesel(table_name = paymentstream)]
+pub struct PaymentStream {
+    // postgres attributed ID for this payment stream
+    pub id: i32,
+    // Account ID of the payer
+    pub account: String,
+    // ID of the payee (msp or bsp)
+    pub provider: String,
+    // Total amount already paid to this provider
```

Comment:
You mean to this provider from this account. If I want total amount for provider I need to get the sum.